src/ic-cdk/src/api/ic0.rs

```diff
@@ -70,45 +70,39 @@ ic0_module! {
     ic0.msg_reply : () -> ();                                                   // U Q Ry Rt
     ic0.msg_reject : (src : i32, size : i32) -> ();                             // U Q Ry Rt
 
-    ic0.msg_funds_available : ( unit_src : i32, unit_size : i32 ) -> i64;       // U Rt Ry Lb
-    ic0.msg_funds_refunded : ( unit_src : i32, unit_size : i32 ) -> i64;        // Rt Ry
-    ic0.msg_funds_accept :                                                      // U Rt Ry
-      ( unit_src : i32, unit_size : i32, amount : i64 ) -> ();
+    ic0.msg_cycles_available : () -> i64;                                       // U Rt Ry
+    ic0.msg_cycles_refunded : () -> i64;                                        // Rt Ry
+    ic0.msg_cycles_accept : ( max_amount : i64 ) -> ( amount : i64 );           // U Rt Ry
 
-    ic0.canister_self_size : () -> (i32);                                       // *
+    ic0.canister_self_size : () -> i32;                                         // *
     ic0.canister_self_copy : (dst : i32, offset : i32, size : i32) -> ();       // *
-    ic0.canister_balance : ( unit_src : i32, unit_size : i32 ) -> i64;           // *
-
-    ic0.ingress_rate_bucket_size : () -> i32;                                   // Ll
-    ic0.ingress_rate_bucket_copy : (dst : i32, offset : i32, size : i32) -> (); // Ll
-    ic0.return_ingress_bucket :
-      ( bucket_src : i32,
-        bucket_size : i32,
-        cost : i64
-      ) -> ();                                                                  // Lb
-    ic0.return_ingress_limit_per_second : (limit : i64) -> ();                  // Ll
+    ic0.canister_cycle_balance : () -> i64;                                     // *
```

Comment:
Surprised to see `ingress_bucket` mentioned here (and removed by you), that has not yet been part of a released spec. Did someone copy from the `master` version of the spec, instead of a suitable release?

Reply:
That is my guess.